test_format/format.eno

```diff
@@ -885,7 +885,7 @@ y 4]
     [potemkin :as p]))
 --Expected
 
-# wrap closers around comments 1
+# slurp closing parens around comments 1
```

Comment:
nit: this should say something like "barf comments 1". Slurping is pulling in a neighbouring form into the current form